assets/src/js/batch.jsx

```diff
@@ -126,7 +130,7 @@ var App = React.createClass( {
 				// Determine if we have to run another step in the batch. Checks if there are more steps
 				// that need to run and makes sure the 'status' from the server is still 'running'.
 				if ( response.currentStep !== response.total_steps && 'running' === response.status.toLowerCase() ) {
-					self.runBatch( currentStep + 1 );
+					self.runBatch( response.current_step + 1 );
```

Comment:
Actually, we need that currentStep variable to be able to pass it back in the AJAX request. The issue is that the server may deincrement the step as a way of running a process again if more have been added. Need to keep both unless there’s a different solution.